dask/JupyterExampleRunthroughOnKubeOnArmP1.py

```diff
@@ -181,19 +193,19 @@ def is_even(x):
 
 
 import dask.bag as db
-b = db.from_sequence(range(1,100))
+b = db.from_sequence(range(1, 100))
 
 
 # In[26]:
 
 
-iseven = lambda x: x % 2 == 0
```

Comment:
@holdenk  I don't think there's a specific reason not to use 'def' construct but lmk if there is.